.gitignore

```diff
@@ -2,11 +2,11 @@
 *.class
 *.balx
 
-# Log file
-*.log
+# Log file (Commented to add ballerina-internal.log)
```

Comment:
Do we need commit the log file?

Reply:
Yes once it is committed will uncomment this line and commit again.

Reply:
We don't need to commit this file.

Reply:
Sure